domain/schema/model/sql/0020-unit.sql

```diff
@@ -23,6 +23,11 @@ CREATE TABLE unit (
     resolve_kind_id INT NOT NULL DEFAULT 0,
     password_hash_algorithm_id TEXT,
     password_hash TEXT,
+    track TEXT,
+    risk TEXT NOT NULL,
+    branch TEXT,
+    os_id TEXT NOT NULL,
+    architecture_id INT NOT NULL,
```

Comment:
suggestion: add a comment here to say that track/risk/branch is for the charm. Platform has a channel also.